src/pages/leejiyon/Components/Nav.scss

```
@@ -0,0 +1,107 @@
+@import '/src/styles/Variables.scss';
+
+.NavJY {
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 54px;
+  width: 100%;
+  padding: 0 83px;
+  background-color: #fff;
+  border-bottom: 1px solid $border-color;
+  z-index: 100;
+
+  .navLogo {
+    position: relative;
+    display: flex;
+
+    img {
+      width: 20px;
+      height: 20px;
+      margin-right: 20px;
+    }
+
+    .navLogoTitle {
+      line-height: 20px;
+      font-size: 20px;
+      font-family: 'Lobster', cursive;
+    }
+  }
+
+  .navLogo .navLogoTitle::before {
+    content: '';
+    position: absolute;
+    left: 30px;
+    top: 50%;
+    transform: translateY(-50%);
+    display: block;
+    width: 1px;
+    height: 20px;
+    background-color: black;
+  }
```

.navLogo 안의 .navLogoTitle 의 가상클래스를 생성하신거면 아래와 같이 하시면 될 것 같습니다.

```suggestion
    .navLogoTitle {
      line-height: 20px;
      font-size: 20px;
      font-family: 'Lobster', cursive;
      &::before {
        content: '';
        position: absolute;
        left: 30px;
        top: 50%;
        transform: translateY(-50%);
        display: block;
        width: 1px;
        height: 20px;
        background-color: black;
        }
      }
```

---

확인했습니다 !